addons/mrp_account/models/mrp_production.py

```diff
@@ -23,6 +23,29 @@ def _compute_show_valuation(self):
         for order in self:
             order.show_valuation = any(m.state == 'done' for m in order.move_finished_ids)
 
+    def _get_detail_extra_cost(self, qty_done):
+        detail_lines = []
+        currency_id = self.company_id.currency_id
+        if not currency_id.is_zero(self.extra_cost):
+            detail_lines.append("Subcontracting cost: %.2f %s/%s - %.2f %s" % (
```

Comment:
you don't know it it's the result of a subcontracting
probably some logic to do in mrp_subcontracting_account